src/utils.js

```diff
@@ -15,16 +17,20 @@
  */
 export function createViewListItemElement( writer ) {
 	const viewItem = writer.createContainerElement( 'li' );
-	viewItem.getFillerOffset = getFillerOffset;
+	viewItem.getFillerOffset = _getFillerOffset;
 
 	return viewItem;
 }
 
 // Implementation of getFillerOffset for view list item element.
 //
 // @returns {Number|null} Block filler offset or `null` if block filler is not needed.
-function getFillerOffset() {
+function _getFillerOffset() {
```

Comment:
```suggestion
function getListItemFillerOffset() {
```

You can be more explicit here and don't force people to understand that there are two functions available here in the scope.